src/locales/en-US/commands/reddit.json

```diff
@@ -1,19 +1,27 @@
 {
 	"name": "reddit",
 	"description": "Get information from Reddit",
+	"subredditName": "subreddit",
+	"subredditDescription": "Get a random post from a specified subreddit",
+	"postName": "post",
+	"postDescription": "Get the information of a specific post",
 	"optionsRedditName": "reddit",
-	"optionsRedditDescription": "The subreddit to get a post from",
-	"invalidName": "Subreddits must be between 3 and 21 characters long, and contain only letters, numbers, or underscores. Additionally, the `/r/` and `r/` prefixes is also supported.",
+	"optionsRedditDescription": "The subreddit name to get a random post from",
+	"optionsPostName": "post",
+	"optionsPostDescription": "A reddit post's link",
+	"invalidName": "Subreddits must be between 3 and 21 characters long, and contain only letters, numbers, or underscores. The `/r/` and `r/` prefixes are also supported.",
+	"invalidPostKey": "Posts must have a key of at least 6 characters, and only container letters and numbers.",
 	"banned": "The specified subreddit is not deemed to be safe, please try another one.",
 	"post": "**{{title}}** submitted by {{author}}\n{{url}}",
 	"noPosts": "There are no posts available in the specified subreddit. Please make sure you have typed the name correctly.",
 	"allNsfw": "All the posts from the specified subreddit are marked as NSFW, which can only be displayed on age-restricted channels.",
 	"allNsfl": "All the posts from the specified subreddit are marked as NSFL, and as such, I cannot show any of them to you.",
 	"abortError": "Reddit didn't respond on time, please try again later.",
+	"parsePostException": "I couldn't parse the data for the specified post.",
 	"unavailableErrorPrivate": "I couldn't get any information from the specified subreddit because it's marked as private.",
 	"unavailableErrorGoldOnly": "I couldn't get any information from the specified subreddit because it's marked as Reddit Premium only.",
-	"unavailableErrorQuarantined": "I couldn't get any information from the specified subreddit because it's quarantined. Reason:\n>>> {{reason}}",
-	"unavailableErrorGated": "I couldn't get any information from the specified subreddit because it's gated. Reason:\n>>> {{reason}}",
+	"unavailableErrorQuarantined": "I couldn't get any information from the specified subreddit because it's quarantined. Reason:\n>>> {{reason}}.",
+	"unavailableErrorGated": "I couldn't get any information from the specified subreddit because it's gated. Reason:\n>>> {{reason}}.",
```

Comment:
```suggestion
	"unavailableErrorQuarantined": "I couldn't get any information from the specified subreddit because it's quarantined. Reason:\n>>> {{reason}}",
	"unavailableErrorGated": "I couldn't get any information from the specified subreddit because it's gated. Reason:\n>>> {{reason}}",
```